packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -556,7 +559,7 @@ abstract class TestRunner {
 
   private doWatchAck(
     ackedTargets: SpecWatchAck,
-    watchSnapshot?: SpecSnapshotVersion
+    watchSnapshot?: SpecWatchSnapshot
```

Comment:
FWIW it seems like we could probably remove "watchSnapshot?" from all of these and just have the test do the watchSnapshot step explicitly as a separate step.  I don't care strongly though.

Reply:
Done